stable/kommander/Chart.yaml

```
@@ -7,4 +7,4 @@ maintainers:
   - name: alejandroEsc
   - name: jimmidyson
 name: kommander
-version: 0.10.3
+version: 0.10.4
```

Let's bump the minor version to reflect adding the airgapped capability:

```suggestion
version: 0.11.0
```